tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.IDataView.cs

```diff
@@ -224,5 +224,19 @@ public void TestIDataViewWithNulls()
             Assert.Equal("", preview.ColumnView[14].Values[5].ToString()); // null row
             Assert.Equal("foo", preview.ColumnView[14].Values[6].ToString());
         }
+
+        [Fact]
+        public void TestDataFrameFromIDataView()
+        {
+            DataFrame df = MakeDataFrameWithAllColumnTypes(10, withNulls: false);
+            IDataView dfAsIDataView = df;
+            DataFrame newDf = DataFrame.FromIDataView(dfAsIDataView);
```

Comment:
What do you think about making an extension method on `IDataView` that returns the `DataFrame`?

```C#
IDataView values = ...;
ITransformer mlModel = ...;

DataFrame df = mlModel.Transform(values).ToDataFrame();
```

vs.

```C#
IDataView values = ...;
ITransformer mlModel = ...;

DataFrame df = DataFrame.FromIDataView(mlModel.Transform(values));
```

Reply:
Yup, you brought it up before I had a chance to. I'm in agreement.